blueocean-dashboard/src/main/js/components/Activity.jsx

```diff
@@ -47,6 +48,14 @@ RunNonMultiBranchPipeline.propTypes = {
 };
 
 export class Activity extends Component {
+    constructor(props) {
```

Comment:
Activity is not the place to do it, see my last comment, that has to go into pipelinePage and passed as children component.

As for now it only works on the activity tab but that should work as well in branch and pr.